server/offenceCodeDecisions/Decisions.test.ts

```diff
@@ -0,0 +1,43 @@
+import decisionTrees from './DecisionTrees'
+
+function findDuplicates<T>(toCheck: Array<T>) {
+  const unique = new Set(toCheck)
+  return toCheck
+    .filter(item => {
+      if (unique.has(item)) {
+        unique.delete(item)
+        return null
+      }
+      return item
+    })
+    .filter(item => item != null)
+}
+
+describe('decisions', () => {
+  it.skip('toString', () => {
+    // Not a test but useful output
```

Comment:
This test is skipped - but if you want to see what the structure looks like then comment this back in